src/classBased/components/TodoContainer.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import TodosList from './TodosList';
+import Header from './Header';
+import InputTodo from './InputTodo';
+
+class TodoContainer extends React.Component {
+  state = {
+    todos: [],
+  };
+
+  // componentDidMount() {
+  //   fetch("https://jsonplaceholder.typicode.com/todos?_limit=10")
+  //     .then((response) => response.json())
+  //     .then((data) => this.setState({ todos: data }));
+  // }
+
```

Comment:
Try to avoid commenting on your code, you should delete all the commented parts before push your code.